openlane/steps/openroad.py

```diff
@@ -828,6 +828,26 @@ def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
         return super().run(state_in, env=env, **kwargs)
 
 
+@Step.factory.register()
+class GlobalPlacementSkipIO(GlobalPlacement):
+    id = "OpenROAD.GlobalPlacementSkipIO"
+    name = "Global Placement Skip IO"
+
+    def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
+        kwargs, env = self.extract_env(kwargs)
```

Comment:
It looks like it will run indiscriminately... it should only run when `matching`?

(Shouldn't hurt too much anyway...)